plugin/virustotal/config.py

```diff
@@ -1,6 +1,6 @@
 
 # VT-IDA plugin global configuration file
 
-API_KEY = ''  # Optional
+API_KEY = ''  # Required for fetching VT Intelligence info, can be public or private API KEY
```

Comment:
Is it still optional?

Reply:
Yes, when the API KEY is not defined it just provides the same functionality than v0.10 (vtgrep integration)